sklearn/linear_model/logistic.py

```diff
@@ -1213,6 +1217,10 @@ def fit(self, X, y, sample_weight=None):
                              self.dual)
 
         if self.solver == 'liblinear':
+            if self.n_jobs != -1:
+                warnings.warn(
```

Comment:
Could you please make the intended edits on top of my branch, just to make sure it follows the convention to make good practice. I thought that would be more efficient.